specificatie/gba-genereervariant/openapi.json

```diff
@@ -297,38 +297,47 @@
   },
   "components" : {
     "schemas" : {
+      "Reisdocumentnummer" : {
+        "pattern" : "^[a-zA-Z0-9]{9}$",
+        "type" : "string"
+      },
       "GbaReisdocument" : {
         "type" : "object",
         "properties" : {
-          "reisdocumentnummer" : {
-            "$ref" : "#/components/schemas/Reisdocumentnummer"
-          },
-          "soortReisdocument" : {
-            "$ref" : "#/components/schemas/Waardetabel"
-          },
-          "autoriteitAfgifte" : {
-            "$ref" : "#/components/schemas/Waardetabel"
-          },
-          "datumUitgifte" : {
-            "$ref" : "#/components/schemas/GbaDatum"
-          },
           "datumEindeGeldigheid" : {
             "$ref" : "#/components/schemas/GbaDatum"
           },
           "datumInhoudingOfVermissing" : {
             "$ref" : "#/components/schemas/GbaDatum"
           },
           "aanduidingInhoudingOfVermissing" : {
-            "$ref" : "#/components/schemas/Waardetabel"
+            "$ref" : "#/components/schemas/AanduidingInhoudingOfVermissing"
           },
           "burgerservicenummer" : {
             "$ref" : "#/components/schemas/Burgerservicenummer"
           },
-          "inOnderzoek" : {
-            "$ref" : "#/components/schemas/GbaInOnderzoek"
+          "overlijden" : {
+            "$ref" : "#/components/schemas/GbaOverlijden"
           }
         },
-        "description" : "Een document dat nodig is voor reizen naar het buitenland.\n* **soortReisdocument** : geeft aan welk Nederlands reisdocument is verstrekt. Omschrijving wordt gevuld op basis van de waarden die voorkomen in de tabel 'Nederlands_Reisdocument' (landelijke tabel 48) uit de Haal-Centraal-BRP-tabellen-bevragen API.\n* **autoriteitAfgifte** : geeft aan welke autoriteit het reisdocument heeft verstrekt. Omschrijving wordt gevuld op basis van de waarden die voorkomen in de tabel 'Autoriteit_Afgifte_Reisdocument' (landelijke tabel 49) uit de Haal-Centraal-BRP-tabellen-bevragen API, waarbij indien van toepassing een gemeentecode (tabel Gemeente) of landcode (tabel Landen) is ingevuld.\n* **datumUitgifte** : de datum waarop het reisdocument is uitgegeven.\n* **datumEindeGeldigheid** : de datum waarop het reisdocument niet meer geldig is.\n* **datumInhoudingOfVermissing**: de datum waarop het reisdocument is vermist, ingehouden, ingeleverd, of van rechtswege is vervallen.\n* **aanduidingInhoudingOfVermissing** : Omschrijving wordt gevuld op basis van de waarden die voorkomen in de tabel 'Inhouding_Vermissing_Reisdocument' uit de Haal-Centraal-BRP-tabellen-bevragen API.\n"
+        "description" : "Gegevens van een reisdocument die nodig zijn om te bepalen of het reisdocument nog geldig en in het bezit van de houder is, plus de identificatie van de houder.\n"
+      },
+      "AanduidingInhoudingOfVermissing" : {
+        "pattern" : "^(I|R|V|\\.)$",
```

Comment:
Opvallend dat hier is gekozen voor een pattern, waarom?

Reply:
omdat dat makkelijk is? Waarom niet?

Reply:
Ik stel het niet ter discussie maar ik probeer achter de beweegredenen te komen.
Je zou hier immers ook een waardetabel kunnen gebruiken net als bij geslacht.

Reply:
geen waardetabel, omdat het gegeven niet aan een eindgebruiker wordt doorgestuurd. Is alleen voor de proxy om informatievragen mee te beantwoorden